frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx

```diff
@@ -22,6 +22,13 @@ export interface CellInfoBarProps {
   cellInfoCellType: Exclude<State["cellInfoCellType"], null>;
   tissueName: string;
 }
+function useHandleHoverEnd(event: EVENTS, hoverStartTime: number, payload = {}) {
+  return useCallback(() => {
+    if (Date.now() - hoverStartTime > 2 * 1000) {
+      track(event, payload);
+    }
+  }, [hoverStartTime]);
+}
```

Comment:
```suggestion 
useHandleHoverEnd = useCallBack((event: EVENTS, hoverStartTime: number, payload = {}) => {
    if (Date.now() - hoverStartTime > 2 * 1000) {
      track(event, payload);
    }
  }, [hoverStartTime])
```

You might need to format that ^